### ISSUE TITLE
Elevated saccharopinuria

### ISSUE THREAD

**Issue by e-lbtn** [obophenotype/human-phenotype-ontology#7083]
**Preferred term label:**
Elevated saccharopinuria

**Definition (free text, please give PubMed ID)**
Presence of saccharopine in the urine
PMID: 33996359_30573525_32768327

**Parent term (use hpo.jax.org/app)**
Abnormal circulating lysine concentration HP:0010908

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Saccharopinuria, ORPHA:3124, OMIM: 268700

**Comment by pnrobinson** [comment]
Note saccharopinuria already implies "elevated".

adding as HP:0034028, the terms will be available with the HPO release in October.
